lib/cli/src/commands/package/tag.rs

```diff
@@ -490,11 +490,17 @@ impl AsyncCliCommand for PackageTag {
             .tag(&client, &manifest, &manifest_path, false, false)
             .await?;
 
-        if !self.quiet {
-            eprintln!(
-                "You can now run your package with `{}`",
-                format!("{} run {id}", bin_name!()).bold()
-            );
+        match id {
+            PackageIdent::Named(ref n) => {
+                eprintln!(
+                    "{} Check out the package's page at {}",
+                    "✔".green().bold(),
+                    format!("https://wasmer.io/{}", n.build_identifier()).bold()
```

Comment:
We need to put the proper url from the host (on this PR).

Ideally, we should fetch the package eurl from the backend, please create an issue for this so we can follow up

Reply:
Should be a function